sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-files/e-cert-file-footer.ts

```diff
@@ -1,18 +1,20 @@
-import { FixedFormatFileLine } from "../../../services/ssh/sftp-integration-base.models";
-import { StringBuilder } from "../../../utilities/string-builder";
+import { FixedFormatFileLine } from "../../../../services/ssh/sftp-integration-base.models";
+import { StringBuilder } from "../../../../utilities/string-builder";
 import {
   ECERT_SENT_TITLE,
-  NUMBER_FILLER,
   RecordTypeCodes,
-  SPACE_FILLER,
 } from "../models/e-cert-full-time-integration.model";
+import {
+  SPACE_FILLER,
+  NUMBER_FILLER,
+} from "../../../models/esdc-integration.model";
 
 /**
  * Footer of an E-Cert file.
  * The documentation about it is available on the document
  * 'CSLP-AppendixF2AsReviewed2016-FileLayouts BC Files V3(HAJ-CB EDITS) In ESDC Folder'.
  */
-export class ECertFileFooter implements FixedFormatFileLine {
+export class ECertFTFileFooter implements FixedFormatFileLine {
```

Comment:
I would suggest avoiding FT/PT as an acronym and using FullTime/PartTime instead. IMO it makes it easier for developers to look at it for the first time.